util.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+func unsetSSHEnvs(sshRsaPrivateKey string) error {
+	for _, env := range os.Environ() {
+		key, value := splitEnv(env)
+
+		if value == sshRsaPrivateKey {
+			log.Debugf("%s has been unset", key)
+			if err := os.Unsetenv(key); err != nil {
```

Comment:
unsetting here is only applicable for the current process. this way the ssh key values are not removed from envman. please wire in envman env delete as well

Reply:
`sshRsaPrivateKey` -> please remove ssh and rsa part of the argument name if the step is only ssh and rsa capable. if there is no other ssh type and security method in the step the these are unneccessary

Reply:
this function `func unsetSSHEnvs(sshRsaPrivateKey string) error` is generic in any way. beacuse you can give it any value, and envs will be unset that contains this value. so using `unsetSSHEnvs` is incorrect here. also `sshRsaPrivateKey` because this argument represents the env's value.